docs/reference/filters.md

```diff
@@ -365,6 +365,32 @@ Parameters:
 The replacement may contain [template placeholders](#template-placeholders).
 If a template placeholder can't be resolved then empty value is used for it.
 
+### normalizePath
+
+Normalize the URL path by removing empty path segments and trailing slashes.
+
+Parameters:
+
+- The URL path
+
+Example:
+
+```
+all: * -> normalizePath() -> "https://backend.example.org/api/v1;
```

Comment:
missing closing `"` and network backend is not having a path segment.
`all: * -> normalizePath() -> "https://backend.example.org";`